### DISCUSSION TITLE
Are there still "sweet spots" for mining for certain ores?

### DISCUSSION THREAD

**Discussion by janimationd** [NillerMedDild/Enigmatica6#1924] [Q&A]
In vanilla there's [the classic guidance that Y levels 11-12 are the sweet spot for mining diamond](https://minecraft.fandom.com/wiki/Diamond#Mining), for instance. I know E6 contains some mods that affect where ore spawns and world generation in general, do minerals still have sweet spots like that or are they evenly distributed throughout the Y levels that they spawn in now?

My experience so far is that mining very low (~ Y level 4) yields more diamonds and other ores than 11/12.

**Comment by NielsPilgaard** [comment]
Ores will generate in higher quantity around their baseline Y level, which is the middlepoint between min/max Y, like Lapis used to be in Vanilla. I hope that explains it well enough :)
